utility/psi_quota.py

```diff
@@ -230,6 +232,31 @@ def get_complete_quota(instance_detail, os_cred, project_name):
     return quota_stats
 
 
+def get_user_names():
+    # URL of the hosted CSV file
+    csv_url = "https://gitlab.cee.redhat.com/rhcs-qe/cephqe-jenkins/-/raw/main/resources/template/rhod_users.csv"
```

Comment:
Can we pass this as a parameter? I would like to avoid hard coding of the filename.